data/events/2022-lagos.yml

```
@@ -0,0 +1,97 @@
+name: "2022-lagos" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2022" # The year of the event. Make sure it is in quotes.
+city: "Lagos" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdayslag" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Lagos!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+cancel: "true"
```

Your event won't appear on the website it this is set to `true`.

```suggestion
cancel: "false"
```